### ISSUE TITLE
too many colons in address

### ISSUE THREAD

**Issue by Astingengo** [oliver006/redis_exporter#357]
I'm having this issue (debug is set to all)
```
exporter_redis.1.52mvtbf05bgt@rhel2    | time="2020-03-13T13:47:11Z" level=debug msg="Trying DialURL(): redis://172.16.222.132:6379,redis://172.16.222.133:6379,redis://172.16.222.134:6379"
exporter_redis.1.52mvtbf05bgt@rhel2    | time="2020-03-13T13:47:11Z" level=debug msg="DialURL() failed, err: invalid database: /172.16.222.133:6379,redis://172.16.222.134:6379"
exporter_redis.1.52mvtbf05bgt@rhel2    | time="2020-03-13T13:47:11Z" level=debug msg="Trying: Dial(): tcp redis://172.16.222.132:6379,redis://172.16.222.133:6379,redis://172.16.222.134:6379"
exporter_redis.1.52mvtbf05bgt@rhel2    | time="2020-03-13T13:47:11Z" level=error msg="Couldn't connect to redis instance"
exporter_redis.1.52mvtbf05bgt@rhel2    | time="2020-03-13T13:47:11Z" level=debug msg="connectToRedis( redis://172.16.222.132:6379,redis://172.16.222.133:6379,redis://172.16.222.134:6379 ) err: dial tcp: address redis://172.16.222.132:6379,redis://172.16.222.133:6379,redis://172.16.222.134:6379: too many colons in address"
```

Redis Exporter is at version v1.3.1

OS Used is RHEL 7.6, IP Forwarding is enabled on all servers ... I build the image from the latest code from github (here) with alpine, and I'm able to ping all 3 IPs.

Also ... when trying to curl from the docker I'm not able (installed it with apk add curl)
```
/ # curl 172.16.222.133:6379

curl: (1) Received HTTP/0.9 when not allowed
```

Curl from outside docker is working
```
[me@rhel1 log]$ curl 172.16.222.133:6379
-ERR wrong number of arguments for 'get' command
-ERR unknown command 'User-Agent:'
```

**Comment by oliver006** [comment]
Have a look at
https://github.com/oliver006/redis_exporter#upgrading-from-0x-to-1x

and

https://github.com/oliver006/redis_exporter#prometheus-configuration-to-scrape-multiple-redis-hosts

TLDR; in redis_exporter 1.x the way you configure monitoring multiple redis instances has changed.

**Comment by Astingengo** [comment]
:-( oh ... so the exporter is not supporting multiple instances :-(

Thanks
